_1327/documents/templates/documents/admin_app_name.html

```diff
@@ -0,0 +1,2 @@
+{% load i18n %}
+{% trans "Documents" %}
```

Comment:
This file is rendered as the app name in Django admin, allowing localization of the app's name.